.azure-pipelines-templates/simulation.yml

```diff
@@ -1,33 +0,0 @@
-jobs:
```

Comment:
Just to clarify, this file and `trace_validation.yml` where unused before this PR (since the switch from ado to gh actions) and they are just being cleaned up?

Reply:
Yes, exactly